scripts/shared.mjs

```diff
@@ -0,0 +1,11 @@
+/** @type {boolean} */
+const supportsUnicodeSets = (() => {
+  try {
+    // eslint-disable-next-line n/no-unsupported-features/es-syntax
+    return new RegExp('', 'v').unicodeSets;
+  } catch {
+    return false;
+  }
+})();
+
+export { supportsUnicodeSets };
```

Comment:
Extracting this because I thought `eslint-plugin-n` should have rule to forbid `assert`, turns out not implemented https://github.com/eslint-community/eslint-plugin-n/issues/59

I already extracted it, so I'm not going to change it back.